### PR TITLE
Upgrade all testing linux CI machines to Ubuntu-20.04

### LINKED ISSUE #3025: Upgrade to a newer version of Ubuntu
As per https://github.com/actions/virtual-environments/issues/3287

We should upgrade to a newer version of ubuntu.

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: .azure-pipelines-k8s-matrix.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ jobs:
           curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
           sudo install minikube-linux-amd64 /usr/local/bin/minikube
           sudo minikube start --driver=none --kubernetes-version $(kubernetes_version)
-          # Although the kube and minikube config files are in placed $HOME they are owned by root
+          sudo mv /root/.kube /root/.minikube $HOME
```
Comment (Nokel81) [inline .azure-pipelines-k8s-matrix.yml:49]: My only concern is that this won't work as expected. I search around but could not find the equivalent to the github-action for azure.

Comment (jim-docker) [reply .azure-pipelines-k8s-matrix.yml:49]: The comment said they are already in `$HOME`. Are they no longer put there, after this minikube install/run is executed on ubuntu 20.04?

Comment (Nokel81) [reply .azure-pipelines-k8s-matrix.yml:49]: According to this run I don't think so...

https://github.com/lensapp/lens/runs/3602487897?check_suite_focus=true

Comment (jim-docker) [reply .azure-pipelines-k8s-matrix.yml:49]: OK, if it works...

**Review by jim-docker — APPROVED** [review]

**Comment by Nokel81** [comment]
@jakolehm Is there a way to run that azure pipeline?

**Review by jakolehm — APPROVED** [review]

**Pull request merged**